apps/linode-marketplace-django/roles/post/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+-
+# motd and creds gen
+- name: copy MOTD template to /etc/motd
+  template:
+    src: templates/motd.j2
+    dest: /etc/motd
+
+- name: writing jitsi creds into file
+  copy:
+    dest: "{{ item.user }}/.credentials"
```

Comment:
I like this idea, but we only document the /home/user/.credentials in the docs shortcode, so I'm not sure if we want to put sensitive creds in both locations without proper documentation. We just restructured this in all ansible guides so personally I would stick to just the home/user dir